### ISSUE TITLE
Create app-like effects

### ISSUE THREAD

**Issue by Djelibeybi** [delfick/photons-interactor#12]
It would be super peachy keen if Interactor could run app-like effects (like Pastel, Animate, Flicker, etc) across multiple devices that don't have their own internal firmware effects.

**Comment by delfick** [comment]
That would be nice.

It's unlikely I'll find time for that any time soon, and it'd be a lot
easier to add when I've finished my tile animation rewrite, release next
version of photons, and then clean up interactor and put it in the photons
repo.

In the meantime, the animations themselves would have to be written as well.

Currently I only have a version of flicker
https://github.com/delfick/photons-core/blob/master/examples/flicker

**Comment by Djelibeybi** [comment]
Right. I figured it needed the new animation rewrite as a base. The ability to send a "stream" of commands where Photons is capable of waiting for the first command to finish prior to actioning the second is required for this, and the issue I logged to turn a device on, transform and off again in a single task too.

**Comment by delfick** [comment]
With my latest round of changes that becomes more true, because I've made the canvas able to understand more than just tiles and create messages for you. Before those changes that was less the case :)

But feel free to try and think of how the animations could behave if you were able to make the most of tile and strip zones.
